README.oracle_fdw

```diff
@@ -255,8 +262,8 @@ Oracle permissions
 The Oracle user will obviously need CREATE SESSION privilege and the right
```

Comment:
Either put that in backticks or (better) remove the backticks from SQL commands and Oracle privileges below.